### ISSUE TITLE
The motivation to design the submodule

### ISSUE THREAD

**Issue by seacj** [sooftware/jasper#2]
It seems that the motivation to design the jasper architecture is not clear.  Most networks are constructed by stacking the basic module such as Resnet, MobileNet. The design of the submodule architecture makes me confused. Could you share the motivation here?

**Comment by kuraga** [comment]
https://arxiv.org/pdf/1904.03288, p. 1:

> As a result, Jasper’s architecture contains only 1D convolution,
> batch normalization, ReLU, and dropout layers – operators
> highly optimized for training and inference on GPUs.
